funcx_endpoint/tests/test_rabbit_mq/test_result_q_heartbeat.py

```diff
@@ -6,11 +6,11 @@
 
 
 @pytest.mark.parametrize("use_heartbeat", [None, 1])
-def test_no_heartbeat(start_result_q_publisher, conn_params, use_heartbeat):
+def test_no_heartbeat(start_result_q_publisher, pika_conn_params, use_heartbeat):
     """Confirm that result_q_publisher does not disconnect when delay
     between messages exceed heartbeat period
     """
-    conn_params = copy.deepcopy(conn_params)
+    conn_params = copy.deepcopy(pika_conn_params)
```

Comment:
Not relevant to the current work, but there's no need to copy a function scoped fixture.
I'm just noticing this now.

Reply:
Heh; observed that, but thought "Not now.  Gotta get by @sirosen's review!"  ;-)

Reply:
At least I'm consistent. 😉